docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -263,6 +263,63 @@ include::{generated-dir}/config/quarkus-vertx-http-config-group-access-log-confi
 |Response header                                                              |          | `%{o,response_header_name}`
 |===
 
+
+[[reverse-proxy]]
```

Comment:
@cescoffier Please update [this OIDC section](https://quarkus.io/guides/security-openid-connect-web-authentication#running-behind-a-reverse-proxy) with a new link

Reply:
Done!